modules/dns/variables.tf

```diff
@@ -0,0 +1,40 @@
+// ----------------------------------------------------------------------------
```

Comment:
I like how the structures are aligning now. I just need to drop the `jx` level now in GKE.